frontend/src/components/search/SearchResultsListItem.tsx

```diff
@@ -111,15 +111,19 @@ export default function SearchResultsListItem({
           </div>
         </div>
         <div className="desktop:grid-col-auto">
-          <div className="overflow-hidden">
+          <div className="overflow-hidden font-body-xs">
             {/* TODO: Better way to format as a dollar amounts */}
-            <span className="display-block desktop:text-right">
+            <span
+              className={`${metadataBorderClasses} desktop:display-block text-right desktop:margin-right-0 desktop:padding-right-0`}
+            >
               <strong>Award Ceiling:</strong>{" "}
-              <span className="desktop:display-block desktop:font-sans-lg text-ls-neg-3">
+              <span className="desktop:display-block desktop:font-sans-lg text-ls-neg-3 text-right">
                 ${opportunity?.summary?.award_ceiling?.toLocaleString() || "--"}
               </span>
             </span>
-            <span className="display-block desktop:text-right">
+            <span
+              className={`${metadataBorderClasses} desktop:display-block text-right desktop:margin-right-0 desktop:padding-right-0`}
```

Comment:
Using the `metadataBorderClasses` so that the award meta gets the responsive styles that put a border between items with they're short enough to appear on one line. The `.text-right` classes only apply on desktop when we override the `display`